cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -123,6 +125,7 @@ static void sc_populate_libgl_with_hostfs_symlinks(const char *libgl_dir,
 						   const char *glob_list[],
 						   size_t glob_list_len)
 {
+	size_t source_dir_len = strlen(source_dir);
```

Comment:
strlen() can be tricky with untrusted inputs, but I verified the callers of this function and all of the inputs are trusted.